web/docs/tutorial/07-auth.md

```diff
@@ -505,7 +498,16 @@ This is it, we have a working authentication system, and our Todo app is multi-u
 
 We did it 🎉 You've followed along with this tutorial to create a basic Todo app with Wasp.
 
-You can find the complete code for the tutorial [here](https://github.com/wasp-lang/wasp/tree/release/examples/tutorials/TodoApp).
+<ShowForJs>
+
+You can find the complete code for the JS version of the tutorial [here](https://github.com/wasp-lang/wasp/tree/release/examples/tutorials/TodoApp).
+
+</ShowForJs>
+<ShowForTs>
+
+You can find the complete code for the TS version of the tutorial [here](https://github.com/wasp-lang/wasp/tree/release/examples/tutorials/TodoAppTs).
+
+</ShowForTs>
```

Comment:
You might have noticed we now have two todo ts apps:
- This one that mimics the tutorial.
- Vince's todo-typescript app.
- 
They're slightly different. Should we keep both or remove one?

Reply:
Let's remove @vincanger 's one? Is there anything we loose by that? If not, let's remove it.